heuristic/repair_operators.py

```diff
@@ -919,17 +946,18 @@ def mip_week_operator_2(  employees, shifts_in_week, competencies, time_periods_
 
     model.setObjective(
     quicksum(
-        quicksum(delta_plus[c, t] + delta_minus[c, t]
-                for t in time_periods_in_week[c, week[0]])
+        quicksum(delta_plus[c, t] + delta_minus[c, t] for t in updated_time_periods)
         for c in competencies
     )
     ,GRB.MINIMIZE,)
 
     #model.setParam("MipFocus", 1)
-    model.setParam("TimeLimit", 10)
+    model.write("Restrictions.lp")
+    model.setParam("TimeLimit", 5)
     model.optimize()
+    model.write("answer.sol")
 
-    shifts = {(t,v): x[t,v].x for t,v in shifts_in_week[week[0]] if x[t,v].x > 0}
+    shifts = {(t,v): x[t,v].x for t,v in shifts_in_week[week[0]] if x[t,v].x > 0.5}
```

Comment:
What is the purpose here?

Reply:
Gurobi does not always set = 1 for shifts that are used. This way we don't end up not using shifts that should be used